test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ValidateTests.cs

```diff
@@ -0,0 +1,185 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Rest;
+using Microsoft.Health.Fhir.Tests.Common;
+using Microsoft.Health.Fhir.Tests.Common.FixtureParameters;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Tests.E2E.Rest
+{
+    [Trait(Traits.Category, Categories.Validate)]
+    [HttpIntegrationFixtureArgumentSets(DataStore.All, Format.All)]
+    public class ValidateTests : IClassFixture<HttpIntegrationTestFixture>
+    {
+        private readonly HttpClient _client;
+        private readonly JsonSerializer _serializer;
+
+        public ValidateTests(HttpIntegrationTestFixture fixture)
+        {
+            _client = fixture.HttpClient;
```

Comment:
Any particular reason we aren't using the `FhirClient` that's exposed test fixture?

Reply:
Mostly because I didn't know about it. Tests updated to use the FhirClient and a new method added to the client for validate requests.